Jumpit/JumpitBaseApi.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  JumpitBaseApi.swift
+//  Jumpit
+//
+//  Created by 이지훈 on 5/22/24.
+//
+import Foundation
+import Moya
+
+// MARK: - Domain
+enum JumpitAPIDomain {
+    case resumes
+    case positions
+}
+
+extension JumpitAPIDomain {
+    var url: String {
+        switch self {
+        case .resumes:
+            return "/resumes"
+        case .positions:
+            return "/positions"
+        }
+    }
+}
+
+protocol JumpitAPIEndpoint: TargetType {
+    var domain: JumpitAPIDomain { get }
+    var urlPath: String { get }
+    var error: [Int: NetworkError]? { get }
+}
+
+extension JumpitAPIEndpoint {
+    var baseURL: URL {
+        return URL(string: PrivacyInfoManager.baseURL)!
+    }
+    
+    var path: String {
+        return domain.url + urlPath
+    }
```

Comment:
```suggestion
protocol JumpitAPIEndpoint: TargetType {
    var path: String { get }
    var error: [Int: NetworkError]? { get }
}

extension JumpitAPIEndpoint {
    var baseURL: URL {
        return URL(string: PrivacyInfoManager.baseURL)!
    }

```
<3번 방식> 분기 처리할 경우의 수가 적다! 싶어서 안하고 싶으시면 이 부분 없애주시고 JumpitAPI에 <3번 방식>으로 함께 바꾸어 주셔도 좋을 것 같습니다!